pkg/resources/validation/cluster/provisioningcluster.go

```diff
@@ -23,6 +23,7 @@ import (
 const globalNamespace = "cattle-global-data"
 
 var mgmtNameRegex = regexp.MustCompile("^c-[a-z0-9]{5}$")
+var fleetNameRegex = regexp.MustCompile("^[-a-z0-9]*$")
```

Comment:
Should this just match the mgmt regex since each cluster usually has one of each, a provisioning and management cluster object with the same name?

Reply:
For RKE2/K3s the management and provisioning cluster objects don't share a name. The provisioning cluster is named based off of what the user inputs into the UI, and the management cluster gets a randomly generated name of the form `c-m-xxxxxxxx`.